test/jdk/javax/swing/plaf/nimbus/PasswordFieldTest.java

```diff
@@ -44,17 +44,20 @@ private static void setLookAndFeel(UIManager.LookAndFeelInfo laf) {
     }
 
     public static void main(String[] args) throws Exception {
-        for (UIManager.LookAndFeelInfo laf :
-                            UIManager.getInstalledLookAndFeels()) {
-            System.out.println("Testing L&F: " + laf.getClassName());
-            SwingUtilities.invokeAndWait(() -> setLookAndFeel(laf));
+        SwingUtilities.invokeAndWait(() -> {
+            for (UIManager.LookAndFeelInfo laf :
+                                UIManager.getInstalledLookAndFeels()) {
+                System.out.println("Testing L&F: " + laf.getClassName());
+                setLookAndFeel(laf);
 
-            System.out.println("Echo char: " +
-                            UIManager.get("PasswordField.echoChar"));
-            if (UIManager.get("PasswordField.echoChar") == null) {
-                throw new RuntimeException(
-                    "PasswordField.echoChar returns null");
+                System.out.println("Echo char: " +
+                                UIManager.get("PasswordField.echoChar"));
+                if (UIManager.get("PasswordField.echoChar") == null) {
+                    throw new RuntimeException(
+                        "PasswordField.echoChar returns null for " +
+                            laf.getClassName());
+                }
```

Comment:
A question of style: do we usually wrap lines before the operator or after? [Java Coding Style](https://www.oracle.com/java/technologies/javase/codeconventions-indentation.html#248) is for the former. This versions seems to have been updated and the code formatting is lost now; [this draft](http://cr.openjdk.java.net/~alundblad/styleguide/index-v6.html#toc-wrapping-lines) is still for the same usage.